ADOPTERS.md

```diff
@@ -1,90 +1,90 @@
-| Organization                                                             | Contact                                                                                                                                                                                                      | Description of Use                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Looks like some changes to this file snuck in 🤔

Reply:
Oh durp. I'll fix.

Reply:
It was edited by prettier... I think. In order to get around this I had to run commit --amend with --no-verify. But its removed from the commit now.